internal/operator-controller/rukpak/bundle/registryv1.go

```diff
@@ -17,3 +20,134 @@ type RegistryV1 struct {
 	CRDs        []apiextensionsv1.CustomResourceDefinition
 	Others      []unstructured.Unstructured
 }
+
+// Get builds a validation schema based on what install modes the operator supports.
+//
+// For registry+v1 bundles, we look at the CSV's install modes and generate a schema
+// that matches. For example, if the operator only supports OwnNamespace mode, we'll
+// require the user to provide a watchNamespace that equals the install namespace.
+func (rv1 RegistryV1) Get() (map[string]any, error) {
```

Comment:
perhaps to use pointer for the receiver?

```suggestion
func (rv1 *RegistryV1) Get() (map[string]any, error) {
```